Add more information about GPU-accelerated tasks (#1482)
QubesOS/qubes-doc: introduction/faq.md

Absolutely.

### Can I run applications, like games and video editors, which require hardware acceleration?

This is possible, but not recommended. It's usually a better idea to have a separate device for GPU-intensive tasks, but if that's not possible for financial or other reasons, this can be done with significant effort and security trade-offs.

Qubes does not implement GPU virtualization, so any GPU-based tasks will require giving a Qube full access to your GPU, so any security vulnerabilities in your GPU will be exploitable if the Qube is compromised. Furthermore, your system will need two GPUs - one to render the Qubes desktop, and one for your GPU-accelerated Qube. If you have an integrated graphics chip on your CPU, you can use the CPU graphics to render Qubes and the dedicated graphics for your GPU-accelerated Qube.

Also note that you will likely experience noticeably worse performance in video games unless you have a good CPU due to the hardening and virtualization features of Qubes.

> For instructions, see [Create a Gaming HVM](https://forum.qubes-os.org/t/create-a-gaming-hvm/19000) on the Qubes forums.

Reviewer: ```suggestion
For instructions, see [Create a Gaming HVM](https://forum.qubes-os.org/t/create-a-gaming-hvm/19000) on the Qubes forum.
```

Author: Fixed

> There are plans to implement GPU virtualization in Qubes, but it will take a while before this is available. See [the GPU acceleration project on GitHub](https://github.com/orgs/QubesOS/projects/17/views/1) for more information.

Reviewer: ```suggestion
There are plans to implement GPU virtualization in Qubes OS, see `the GPU acceleration project on GitHub <https://github.com/orgs/QubesOS/projects/17/views/1>`__ for more information.
```
